recipes/singularity-compose/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "singularity-compose" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94eb17bab234d9c25afbee92970adc4e9400cf70dac9da6797b84e91974a09de
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - singularity-compose=scompose.client:start
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - pytest-runner
+  run:
+    - python >=3
+    - requests
+    - pyaml
+    - spython
```

Comment:
In that case do you want to do this?
```suggestion
    - spython >=0.0.67
```
It wasn't available earlier as there was an issue with the CDN mirroring (see the Gitter chat).